### DISCUSSION TITLE
[Question] soa_vector getting the max_value for a member

### DISCUSSION THREAD

**Discussion by matthijs** [jfalcou/eve#2001] [Q&A] [answered]
In short, I have a soa_vector with a user defined struct. And now I want to fetch the max_value for a the second member in that struct.

Minimal example of what I am trying to achieve: https://godbolt.org/z/dxPbPfrx5

This does not sound to hard to achieve but I am missing a crucial step it seems...

**Comment by jfalcou** [comment, marked as answer]
You can pass a predicate as a 2nd member of max_value in which you can compare what you want

https://godbolt.org/z/6hWWc816d

**Reply by matthijs** [reply]
Thanks!

**Reply by jfalcou** [reply]
You're welcome !
